manifests/m/MobileStream/EasyTether/1.3.4/MobileStream.EasyTether.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: MobileStream.EasyTether
+PackageVersion: 1.3.4
+PackageLocale: en-US
+Publisher: Mobile Stream
+PackageName: EasyTether
+License: Unknown
+Copyright: Copyright (c) Mobile Stream. All rights reserved.
+ShortDescription: EasyTether

```

Comment:
Please provide a better short description

Reply:
Should be updated now